src/features/goblins/trader/selling/components/Drafting.tsx

```diff
@@ -107,13 +115,24 @@ export const Drafting: React.FC<DraftingProps> = ({
         </div>
 
         <div className="flex flex-col pt-4 pb-3">
-          <h1 className="mb-4">Trade Details</h1>
+          <h1
+            className={classNames("mb-4", {
+              "opacity-50": !hasItemsToList,
+            })}
+          >
+            Trade Details
+          </h1>
           <div className="flex items-start justify-between mb-2">
-            <div className="relative w-full mr-4">
+            <div
+              className={classNames("relative w-full mr-4", {
+                "opacity-50": !hasItemsToList,
+              })}
+            >
               <input
                 type="number"
                 name="resourceAmount"
-                value={resourceAmount}
+                value={resourceAmount.toString()}
```

Comment:
Can you please explain the reason behind converting the value to string?

Reply:
Thanks for catching that, it was testing code that I forgot to remove.  This has been removed now.